core/tests/protractor_desktop/embedding.js

```diff
@@ -258,7 +258,9 @@ describe('Embedding', function() {
         by.css('.protractor-test-embedded-exploration > iframe')));
       waitFor.pageToFullyLoad();
 
-      // TODO(kevinlee12): Remove this sleep.
+      // TODO(#7622): Remove this sleep. This sleep was placed here as a
```

Comment:
Could you please add, to this TODO: "Remove the TODO in scripts/pre_commit_linter.py pointing to the same issue"?

Reply:
done!